source/getting_started/concepts.rst

```diff
@@ -1,120 +1,44 @@
-Concepts
+Key Concepts
 ========
 
+Why is Howso Engine AI that you can understand, trust, and control?  Howso is built on fundamentally different technology from today's black box AI, which enables a variety of new capabilities. Below, we review some of
```

Comment:
"why is"? the grammar here sounds really strange

Reply:
I can see that. Changed to "What makes the Howso Engine AI that you can understand, trust, and control?"